x-pack/plugins/lens/server/routes/existing_fields.ts

```diff
@@ -142,6 +142,8 @@ export function buildFieldList(indexPattern: IndexPattern, metaFields: string[])
   });
 }
 
+type LensSearchResponseType = Array<{ fields: Record<string, unknown[]>; [key: string]: unknown }>;
```

Comment:
@delvedor @mshustov I am still waiting for `LazyDocument` to be fixed as a result of my previous round of comments: ﻿https://github.com/elastic/kibana/pull/83808#issuecomment-771852520

Reply:
@wylieconlon can it be done in a follow-up? the same question for https://github.com/elastic/kibana/pull/83808#discussion_r599768344

Reply:
@mshustov it can be done in a follow-up, yes

Reply:
Actually, both of these issues have been solved. Maybe something went wrong during the last sync?
`LazyDocument` no longer exist and `ruintime_mappings` is defined [here](https://github.com/elastic/elasticsearch-js/blob/type-definitions/api/types.d.ts).

Reply:
@wylieconlon could you elaborate on this https://github.com/elastic/kibana/pull/83808#issuecomment-769976078 ?

Reply:
@mshustov I already gave a full reply here: https://github.com/elastic/kibana/pull/83808#issuecomment-771001843

Reply:
@mshustov I already gave a full reply here: https://github.com/elastic/kibana/pull/83808#issuecomment-771001843

Reply:
This should no longer be a problem, as the current types (see [here](https://github.com/elastic/elasticsearch-js/blob/8b784b217f2e0320359c7b135bee9b8518b71d52/api/types.d.ts#L6405) define `fields` as `Record<string, any>`.
In a future iteration, we could do `Record<string, any[]>`.

Reply:
@wylieconlon awesome, I missed it.